homeassistant/components/comed_hourly_pricing/sensor.py

```diff
@@ -38,12 +38,12 @@
     SensorEntityDescription(
         key=CONF_FIVE_MINUTE,
         name="ComEd 5 Minute Price",
-        native_unit_of_measurement="c",
+        native_unit_of_measurement="c/kWh",
```

Comment:
Please use our common unit constants that apply.

https://github.com/home-assistant/core/blob/b54fe14a10a7a0ee3762c217c32cc6582ace406c/homeassistant/const.py#L505

https://github.com/home-assistant/core/blob/b54fe14a10a7a0ee3762c217c32cc6582ace406c/homeassistant/const.py#L488